OpenNefia.Content/CharaMake/CharaMakeLogic.cs

```diff
@@ -0,0 +1,88 @@
+﻿using OpenNefia.Core.IoC;
+using OpenNefia.Core.Log;
+using OpenNefia.Core.UserInterface;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.CharaMake
+{
+    public interface ICharaMakeLogic
+    {
+        void RunCreateChar();
+    }
+    public class CharaMakeLogic : ICharaMakeLogic
+    {
+        [Dependency] private readonly IUserInterfaceManager _uiManager = default!;
+
+        public List<ICharaMakeLayer> GetDefaultCreationSteps()
+        {
+            return new List<ICharaMakeLayer>
+            {
+                new CharaMakeRaceSelectLayer(),
+                new CharaMakeGenderSelectLayer(),
+                new CharaMakeClassSelectLayer(),
+                new CharaMakeAttributeRerollLayer(),
+                // TODO add feat selection
+                new CharaMakeAliasLayer(),
+                // TODO add appearance
+                // TODO add character sheet
+            };
+        }
+
+
+        public void RunCreateChar()
+        {
+            var steps = GetDefaultCreationSteps();
+            var data = new CharaMakeData();
+            var stepIndex = 0;
+            var step = CharaMakeStep.Continue;
+
+            while (step != CharaMakeStep.Cancel)
+            {
+                if (stepIndex >= steps.Count)
+                {
+                    //just for testing
+                    Logger.Debug($"Character creation complete, values:" + Environment.NewLine 
```

Comment:
Nitpick: Prefer `Logger.DebugS("charamake", $"<...>")`. This lets the logging system toggle log levels on and off for specific engine concerns.

Reply:
done. 76f6762